scripts/install

```diff
@@ -27,14 +28,17 @@ final_path=/var/www/$app
 sudo mkdir -p $final_path
 extract_source $final_path
 
-# Files owned by root, www-data can just read
+# Create system user dedicace for this app
+ynh_system_user_create $app
+
+# Files owned by user specific can just read
 sudo find $final_path -type f | xargs sudo chmod 644
 sudo find $final_path -type d | xargs sudo chmod 755
-sudo chown -R root: $final_path
+sudo chown -R $app: $final_path
 
 # except for data and tmp subdir, where www-data must have write permissions
 sudo mkdir -p $final_path/{data,tmp}
-sudo chown -R www-data:root $final_path/{data,tmp}
+sudo chown -R $app:root $final_path/{data,tmp}
```

Comment:
If you need write permission in those directories. I guess you don't need it on another directories.
So keep, at the line 37, root as owner of the main directory.
Your specific user have a read access, it's enough.